.github/workflows/standardrb.yml

```diff
@@ -0,0 +1,35 @@
+name: StandardRB
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: ["**"]
+
+jobs:
+  standard:
+    name: StandardRB Check Action
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ["2.7"] # use minimal supported version and those new ones, which will might have some breaking changes
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Ruby ${{ matrix.ruby-version }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+      # - uses: actions/cache@v1
+      #   with:
+      #     path: vendor/bundle
+      #     key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+      #     restore-keys: |
+      #       ${{ runner.os }}-gems-
+      - name: Bundle install
+        # We have to stay with bundler version 2.4.22, because of ruby 2.7 requirement, latest working version there
+        run: |
+          gem install bundler -v 2.4.22
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
```

Comment:
I don't think we want to install all the gems for the second time. It leads to long running CI. We already install the deps in Ruby workflow by running `dip provision`. I guess we could call `dip standardrb` in a new step. What do you think?

Reply:
Definetely not my cup of tea :)

I think, each type of spec/test/lint/build/etc. should have each own step in CI.

We install gems for each version of ruby listed anyway, so its not second time like you say — but.. sixth? And it also parallelized, so I dont see practical win in performance — CI will last long (~1m) either way. Doesn't look like a problem for me :)

Reply:
@bibendi sorry to bother, but its being long time since last communication we got here. So, are we okay with current implementation of the job?

If not, I have another variant — I may actually just delete this entire workflow if we can't agree on the topic so it would be easy to merge the branch now :)

Reply:
Yeah, could you just delete this workflow? I'll think later how to implement it on my own

Reply:
Okay, I've forcepushed my latest commit without standardrb workflow — just bump ruby version in standardrb.yml and autofixes with it :)